vfs/dir.go

```diff
@@ -418,6 +422,15 @@ func (d *Dir) rename(newParent *Dir, fsDir fs.Directory) {
 	}
 }
 
+// convert path to name
+func name(p string) string {
+	p = path.Base(p)
+	if p == "." {
+		p = "/"
```

Comment:
That should probably be `p = ""` to fit with rclone conventions, not 100% sure!

Reply:
I think I copied this from here:

https://github.com/rclone/rclone/blob/e9e9feb21eeb200782180e8532f2a0a95f745740/vfs/dir.go#L163-L172